controller_interface/include/controller_interface/internal/robothw_interfaces.h

```diff
@@ -63,7 +63,7 @@ inline std::string enumerateElements(const T&           val,
 }
 
 
-template <typename T>
+template <class T>
```

Comment:
I guess, you changed this for consistency reasons?

Reply:
To be honest, I don't remember the logic for this change, but I don't think it matters; I'm happy for us to consistently use `typename` or `class`, and can revert this part of the change if the decision is for `typename`. I think the only time it matters, `typename` is preferred (eg in ab14ea48d73a1037f79686cf64f3379f9eac0a92).

Reply:
> and can revert this part of the change if the decision is for typename

The other templates use `class` in their definitions as well, so it is fine for me.
`class` seems to be mandatory for nested templates (C++14 and older).

Reply:
No strong opinion on this, do it at your leasure.